### PR TITLE
creating a series of targets to shoot

### PR DESCRIPTION
**Pull request by vkarinje**

### CODE REVIEW

File: blankwindow.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 bg_surf=pygame.image.load('./graphics/background.png').convert()
 
 #importing text
-font=pygame.font.Font('./graphics/subatomic.ttf', 50)
+font=pygame.font.Font('../graphics/subatomic.ttf', 50)
```
Comment (AkhilaChiluveru) [inline blankwindow.py:16]: The file path seems to be incorrect. As classes and graphics are in one directory, we need not add .. two dots before finding the graphics folder. Please revise all of the paths for targets,bullets,background,and font.

File: classes/Targets.py [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+        self.image=pygame.image.load('../graphics/targets.png').convert_alpha()
         self.rect=self.image.get_rect(center=positon)
+
+        self.pos=pygame.math.Vector2(self.rect.topleft)
+        self.direction=pygame.math.Vector2(uniform(-0.5,0.5),1)
+        self.speed=randint(400,600)
+    def update(self,dt):
+        self.pos+=self.direction * self.speed * dt
+        self.rect.topleft=(round(self.pos.x),round(self.pos.y))
+
```
Comment (AkhilaChiluveru) [inline classes/Targets.py:18]: This code is looking good to me. But took a while to get it understood. Adding comments will be very much helpful.

**Comment by AkhilaChiluveru** [comment]
Your underlying code changes that actually create a series of targets took well to me other than the graphics relative path in code. However, there is another concern. please remove pycache folder that is also pushed to the repo. You can achieve this by creating a .gitignore file or by not adding all files with git add . command.

File: blankwindow.py (hunk 1 above)
Comment (vkarinje) [reply blankwindow.py:16]: Adding . one dot is giving me an error. I am adding two dots here as we are going one folder up. Classes and graphics are both in the same directory. But I add ..dots to go one folder up into the graphics folder from the classes folder

File: classes/Targets.py (hunk 2 above)
Comment (vkarinje) [reply classes/Targets.py:18]: Adding comments for understanding of the code.

**Comment by AkhilaChiluveru** [comment]
Code changes look good to me.

**Pull request closed without merging**